_includes/header.html

```diff
@@ -11,6 +11,31 @@
         </svg>
       </a>
 
+      {% include navbar_search.html %}
+
+      <div class="dropdown">
+        <a class="page-link" href="{{site.url}}">Plugins</a>
+        <div class="dropdown-content">
+          <a href="{{ "/extractors/" | prepend: site.baseurl | prepend: site.url }}">Extractors</a>
```

Comment:
```suggestion
          <a href="{{ "/extractors/"  }}">Extractors</a>
```

all of these prepends can be dropped